### ISSUE TITLE
Error installing on macOS

### ISSUE THREAD

**Issue by nicholasc** [audreyt/node-webworker-threads#183]
Node version: 10.4.0

```
../src/WebWorkerThreads.cc:205:3: error: reference to 'ArrayBufferAllocator' is ambiguous
  ArrayBufferAllocator a;
  ^
../src/ArrayBufferAllocator.h:3:7: note: candidate found by name lookup is 'ArrayBufferAllocator'
class ArrayBufferAllocator : public v8::ArrayBuffer::Allocator {
      ^
/Users/nicholas/.node-gyp/10.4.0/include/node/node.h:217:7: note: candidate found by name lookup is
      'node::ArrayBufferAllocator'
class ArrayBufferAllocator;
      ^
../src/WebWorkerThreads.cc:207:32: error: use of undeclared identifier 'a'
  cp.array_buffer_allocator = &a;
                               ^
```
Let me know if you need more information...

**Comment by kraifpatrik** [comment]
I've got the same problem on Ubuntu 16.04.

**Comment by nicholasc** [comment]
I understand that is due to a conflict between the ArrayBufferAllocator in the library and the one defined in node headers. 

@audreyt what would be your approach to solve this? 

Namespace the ArrayBufferAllocator class in the library and access it explicitly? If that is the case I don't mind making a quick PR

**Comment by pohi8056** [comment]
Got the same problem on MacOS

**Comment by nicholasc** [comment]
I have the PR ready but I'm waiting for @audreyt to confirm the namespace adoption. @kraifpatrik @pohi8056, you can use my branch while waiting for this to get resolved.

https://github.com/nicholasc/node-webworker-threads/tree/fix-arraybufferallocator

**Comment by audreyt** [comment]
Released as 0.7.14
